README.md

```diff
@@ -1,24 +1,171 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)ru
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 Capstone - i-model-cars <a name="about-project"></a>
 
-* Configuration
+**Ruby on Rails**
 
-* Database creation
+> i-model-cars app helps `users` to book/make a reservation for latest model cars. It allows user to log in with `username` and `e-mail` and then reserve, add/create and delete cars. It is built on Rails API with React using `Rubygem` and `npm` as software packages system.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+> `Ruby 3.1.3` > `postgresql` > `Rails 7` > `React`
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+<summary>Backend/Frontend</summary>
+  <ul>
+    <li><a href="https://www.rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.react.org/">Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Car Models**
+- [ ] **Reservations**
+- [ ] **API**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Link](https://i-model-cars.onrender.com/api/v1)
+- [Link to frontend repo.](https://github.com/GordonOtieno/i-model-cars-frontend)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Desktop or Laptop Computer running on with Windows, Mac OS or Linux operating system.
+- Rails installed on the Computer
+
+### Setup
+
+Clone these repositories to your desired folder:
+
+- `https://github.com/GordonOtieno/i-model-cars-backend.git` and 
+- `https://github.com/GordonOtieno/i-model-cars-frontend.git` if you prefer ssh and hit enter.
+- create a directory in your preferred location (e.g in the Desktop or Document directory)
+
+### Install
+
+Install this project with:
+
+- Install Ruby on Rails framework.
+- Use RubyGems as a software package system.
+- Dependencies _(a packet manager is required)_: then you run the command `bundle install` in the terminal
+
+### Usage
+
+To run the project, execute the following command:
+
+- `rails server`
+- This will run the project in the terminal in the specified directory
```

Comment:
- [ ] Kindly, include in your instructions the db commands that your users will need to run the project: `rails db:create, reails db:migrate, rails db:seed`.

That way, your users will be able to run the project smoothly.  👍

Reply:
Implemented changes.